roles/openshift_logging/tasks/main.yaml

```diff
@@ -12,9 +12,10 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
-- name: Ensuring ruamel.yaml package is on target
-  command: yum install -y ruamel.yaml
-  check_mode: no
+- name: Ensuring python2-ruamel-yaml package is on target
+  package:
```

Comment:
I was hoping to write up a quick pre-requisites playbook and move this along with PyYAML and other openshift_facts dependencies there.